src/includes/performance/distributed-tracing/java.mdx

```diff
@@ -2,18 +2,35 @@
 
 Traces can span across multiple software services. Each span can be represented as a `sentry-trace` header, containing the trace id, span id and sampling details. This `sentry-trace` header, can be passed to downstream services so that they can create spans being a continuation of the trace started in the originating service.
 
-To obtain a trace header from the span use `ISpan#toTraceHeaders()` method. Then pass it to the downstream service. If the communication happens over HTTP it is recommended to set the value to the `sentry-trace` (also available as a constant `SentryTraceHeader.SENTRY_TRACE_HEADER`) HTTP header.
+To obtain a trace header from the span use `ISpan#toSentryTrace()` method. Then pass it to the downstream service. If the communication happens over HTTP it is recommended to set the value to the `sentry-trace` (also available as a constant `SentryTraceHeader.SENTRY_TRACE_HEADER`) HTTP header.
 
 To create a span as a continuation of the trace retrieved from the upstream service, pass the `sentry-trace` header value to the transaction context:
 
 ```java
+import io.sentry.Sentry;
+import io.sentry.SentryTraceHeader;
+import io.sentry.TransactionContext;
```

Comment:
missing import for `ITransaction` and `InvalidSentryTraceHeaderException`
please always be sure that the code snippets compile in an empty Java class.

here is the reason why https://develop.sentry.dev/sdk/philosophy/#assume-novice-developers

the idea is that a simple copy-paste would work

Reply:
In this case the example will not compile because we assume that `request` is already there and it's an instance of `HttpServletRequest`. Should we expand this example to the full servlet class?

Reply:
not really, it's fine to assume `request` is available thru a different lib or method param, we need to be careful about our own bits because the code snippet is mainly about the Sentry SDK itself.

Reply:
Ok, now all the missing imports are there.